pages/credits.md

```diff
@@ -1,12 +1,25 @@
+[Workflow](https://michaelrappa.github.io/usa-migration/pages/workflow.html) |
+[Credits](https://michaelrappa.github.io/usa-migration/pages/credits.html) |
+[Bibliography](https://michaelrappa.github.io/usa-migration/pages/bibliography.html) |
+[Sources](https://michaelrappa.github.io/usa-migration/pages/sources.html)
+
+
 # Credits
 
 ---
 
 ## Project Credits
 
-* Put your group member's credits here
-* Link to any emails or github accounts (if you want)
-* Leave the credits at the bottom
+* Michael Rappa
+    * My role in the project will be in the position of the Project Manager. I will make sure everything in the project goes smoothly by facilitating communication within the group, controlling the GitHub, and by making decisions if we reach an impasse at any point in the project. The parts of the project that I will be responsible for writing will be the “introduction” and the “next steps”. In addition to the more formal responsibilities I am taking, I will continue to provide guidance to the group in accordance with what our overarching theme is, and will try to project that vision onto the project at all levels. The help I will need from the group would be their continued enthusiasm and teamwork towards completing the project, as well as regular updates on things like progress, problems, ideas, etc.
+* [Thomas Miller](mailto:tmiller26@mail.usf.edu)
+    * My role in the project will be the Digital Scholar. I will be responsible for the sources we research on our project and ensure they are from credible accounts. I will record the use of data by writing a "digital methods and historic scholarship" page. This page will inform readers on how we used the research and what research we had ended up using. There will be times where we find data or research and the information may be useful, but not towards the goal of the project and will be noted in the section. I will plan on using the sources to inform the other members on how best to look at them for intended data that will be useful for the project. From the data I will look for data files and any other useful materials that I can send to my team for visuals and noticing trends for our project.
+* Faris Chaoui
+    * The role that I have in this project is a mixture of both Data Scientist and Visualization expert. I will do both because some of our sources give us the data we need that we do not even have to really scrape. I personally need to accomplish by executing the roles I have as perfect as possible to make this project look good. I intend to get there by using the sources that I have to make this project as presentable as not only but also my group would like. The tools I plan to really use more is tableau since that helps the most with my visualizations. If I need help with my roles to clear my confusions, I will ask my group members but mostly I plan on using the knowledge that I have acquired in class to help. 
 
 ---
```

Comment:
Now that the project is finished, you all might update this page if you want to leave it for posterity. First, you should change it to past tense, since you have accomplished it now! And now, you might be able to easily change it a bit to more clearly talk about what you each accomplished, what pages you were in charge of, et.c.